src/App.tsx

```diff
@@ -1,14 +1,44 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todoIsLoaded, setTodoisLoaded] = useState(false);
+  const [modalIsOpened, setModalIsOpened] = useState(false);
```

Comment:
can you check if the modal should be opened by `currentTodo`? if `currentTodo` has the title or the right id you can automatically show the modal